test/pte_longtest_2phaseVinetSn.hpp

```diff
@@ -0,0 +1,135 @@
+//------------------------------------------------------------------------------
+// © 2021-2024. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract 89233218CNA000001
+// for Los Alamos National Laboratory (LANL), which is operated by Triad
+// National Security, LLC for the U.S.  Department of Energy/National
+// Nuclear Security Administration. All rights in the program are
+// reserved by Triad National Security, LLC, and the U.S. Department of
+// Energy/National Nuclear Security Administration. The Government is
+// granted for itself and others acting on its behalf a nonexclusive,
+// paid-up, irrevocable worldwide license in this material to reproduce,
+// prepare derivative works, distribute copies to the public, perform
+// publicly and display publicly, and to permit others to do so.
+//------------------------------------------------------------------------------
+
+#ifndef _SINGULARITY_EOS_TEST_PTE_TEST_2PHASEVINETSN_
+#define _SINGULARITY_EOS_TEST_PTE_TEST_2PHASEVINETSN_
+
+#include <stdlib.h>
+
+#include <ports-of-call/portability.hpp>
+#include <ports-of-call/portable_arrays.hpp>
+#include <singularity-eos/eos/eos.hpp>
+
+constexpr int NMAT = 2;
+constexpr int NTRIAL = 20;
+constexpr int NPTS = NTRIAL * NMAT;
+constexpr int HIST_SIZE = 10;
+
+constexpr Real in_rho_tot[NTRIAL] = {
```

Comment:
For future us it might be useful to have a comment where these numbers come from. Are they from an equivalent test in flag or something like that?

Reply:
Thanks for adding said comment. :)